io.github.yamanqd.antware.yml

```diff
@@ -0,0 +1,82 @@
+app-id: io.github.yamanqd.antware
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run_antware.sh
+appdata-license: gpl-3.0
+
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+  - name: SFML
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+        -DSFML_BUILD_EXAMPLES=0
+        -DSFML_BUILD_DOC=0
+        -DSFML_BUILD_AUDIO=1
+        -DSFML_BUILD_WINDOW=1
+    post-install:
+      - cp /app/pkgconfig/*.pc $FLATPAK_DEST/lib/pkgconfig/
+    sources:
+      - type: git
+        url: https://github.com/SFML/SFML.git
+        tag: 2.6.0
+        commit: 11b73743c42cf7ecd7c596ba83fdbf1150ffa96c
+        disable-shallow-clone: false
```

Comment:
What's the reason for disabling shallow clone everywhere?

Reply:
it's actually enabled because disable is false. I'd suggest just removing the lines.

Reply:
Ah I said the opposite

Reply:
> What's the reason for disabling shallow clone everywhere?

Figured it would reduce cloning size, not sure what is its default value so I set it false.

Reply:
```suggestion
```

Reply:
https://github.com/SFML/SFML/blob/master/CMakeLists.txt#L239

This is where the bug is. `SFML_PKGCONFIG_DIR` seems to not be defined.

I would submit a patch.